pkg/util/common.go

```diff
@@ -22,6 +24,10 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/version"
 )
 
+func init() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Do those in tests that need them instead

Reply:
The exponential backoff jitter needs this.